src/modules/roc_core/target_libunwind/roc_core/backtrace.cpp

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#define UNW_LOCAL_ONLY
+#include <libunwind.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "roc_core/backtrace.h"
+
+/* Function to convert unsigned int to decimal/hex string.
+ * This function is signal safe.
+ * Parameters :
+ * number = the number to be converted.
+ * base = 10 for decimal to char* and 16 for hex to char* conversion.
+ * output_string = the decimal/hex string
+ */
+
+namespace {
```

Comment:
Nitpick: please move the anonymous namespace inside roc::core and add a trailing "// namespace" comment, like this:

```
namespace roc {
namespace core {

namespace {

...

} // namespace

...
```

(just to be inline with the code-style)

Reply:
The same for the bionic version.